UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
Indeed, your branch seems to be based on a commit _after_ I added the SwiftLint script in the main target… but _before_ I decided to split that script in two (one part in the `swiftgen-cli` target to only check the swiftlint + GenumKit codebase, another in the `UnitTests` target to test the templates generated code).

That's why you have SwiftLint running only when Building but not when running tests, while as I now decided to split them in the 2 targets since that commit you branched from, I now test the templates when doing a `Cmd-U` instead of a `Cmd-B`.

That doesn't explain why you don't have any `force_cast` warning though. Might be worth checking it warns properly once you'll have rebased your branch, and investigate why it doesn't catch those if you still don't have any warning about them when in `master`.